src/main/java/org/broadinstitute/consent/http/models/LibraryCard.java

```diff
@@ -1,6 +1,8 @@
 package org.broadinstitute.consent.http.models;
```

Comment:
I think it's worth testing this class in more detail, but we can push that into a separate PR.